keras_cv/bounding_box/utils.py

```diff
@@ -87,3 +87,110 @@ def _format_outputs(boxes, squeeze):
     if squeeze:
         return tf.squeeze(boxes, axis=0)
     return boxes
+
+
+def pad_with_sentinels(bounding_boxes):
+    """Pads the given bounding box tensor with -1s.
+
+    This is done to convert RaggedTensors to be converted into
+    standard Dense tensors, which have better performance and
+    compatibility within the TensorFlow ecosystem.
+
+    Args:
+        bounding_boxes: a ragged tensor of bounding boxes.
+        Can be batched or unbatched.
+
+    Returns:
+        a Tensor containing the -1 padded bounding boxes.
+    """
+    return bounding_boxes.to_tensor(-1)
```

Comment:
lets use the keyword argument here instead of positional.

default_value=-1.

Reply:
added.